exhibition/urls.py

```diff
@@ -14,8 +14,12 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', include('exhibitor.urls')),
 ]
+
```

Comment:
unnecessary extra lines, you can auto format your code by tools like https://github.com/psf/black